node/tailwind.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  future: {
+    // defaultLineHeights: true,
+    standardFontWeights: true,
+    // removeDeprecatedGapUtilities: true,
+    // purgeLayersByDefault: true,
+  },
+  purge: ["./views/**/*.hbs"],
+  theme: {
+    borderColor: (theme) => ({
+      ...theme("colors"),
+      default: theme("colors.gray.200"),
+    }),
+    container: {
+      padding: {
+        default: "1rem",
+        sm: "2rem",
+        lg: "4rem",
+        xl: "5rem",
+      },
+    },
+    extend: {},
+  },
+  variants: {},
+  plugins: [],
+  experimental: "all",
+};
```

Comment:
in two words what do we customize compared to all-default tailwind?

Reply:
Sadly it's not easy to track with all the releases, i just asked Adam, but a lot of stuff that i need:
- Better colors
- Default line-height
- Extra spacing utilities (for ex h-0.5 or w-96)
- @apply for complex classes (@apply hover:bg-red-200)